helm/charts/determined/README.md

```diff
@@ -24,6 +24,7 @@ The Determined application comes pre-configured with most intallations values to
 | Helm Chart Config Value  | Description |
 | ------------- | ------------- |
 | Region | Region where you are deploying determined. This should match the region where you intend to execute most of your training workloads.  |
+| K8s Scheduler | Default Scheduler type for experiments and notebooks, can be overriden in experiment configuration. (default: default-scheduler) |
```

Comment:
| Scheduler | Default Scheduler type for experiments and notebooks, can be overriden in experiment configuration. Default scheduler is good for most use cases, however large distributed training jobs will want to use coscheduler to allow jobs to queue when resources to run all jobs in parallel are not available. (default: default-scheduler) |